ibm/mas_devops/roles/entitlement_key_rotation/README.md

```diff
@@ -0,0 +1,117 @@
+entitlement_key_rotation
+===============================================================================
+
+This role creates/updates the MAS username and password credentials throughout all MAS related namespaces for one or multiple clusters.
```

Comment:
Can we update this line (and the similar line in the PR description) as it sounds like it might update a MAS User (as in a actual user setup in MAS rather than the infrastructure user to pull secrets). I could see that this might be misunderstood as to what this role does. Maybe add that this is the username/password used to pull images?

Reply:
Added something as:
```
This role creates/updates the entitlement username and password that are stored in the secrets used to pull images throughout all MAS related namespaces for one or multiple clusters.
```